documentation/modules/creating-form-based-storage-maps-ui-vmware.adoc

```diff
@@ -32,11 +32,9 @@ You can create ownerless storage maps by using the form page of the {project-sho
 * *Storage secret*: Select from the list.
 * *Storage product*: Select from the list.
 +
-[IMPORTANT]
+[NOTE]
 ====
-Storage copy offload is Developer Preview software only. Developer Preview software is not supported by Red{nbsp}Hat in any way and is not functionally complete or production-ready. Do not use Developer Preview software for production or business-critical workloads. Developer Preview software provides early access to upcoming product software in advance of its possible inclusion in a Red{nbsp}Hat product offering. Customers can use this software to test functionality and provide feedback during the development process. This software might not have any documentation, is subject to change or removal at any time, and has received limited testing. Red{nbsp}Hat might provide ways to submit feedback on Developer Preview software without an associated SLA.
-
-For more information about the support scope of Red{nbsp}Hat Developer Preview software, see link:https://access.redhat.com/support/offerings/devpreview/[Developer Preview Support Scope].
+Storage copy offload is a feature that allows you to migrate {vmw} virtual machines (VMs) that are in a storage array network (SAN) more efficiently. The feature makes use of the `XCOPY` command in {vmw}'s vSphere Storage APIs-Array Integration (VAAI). This command lets you copy data inside a SAN more efficiently than copying the data over a network. Storage copy offload is available as a Technology Preview feature for {project-first} 2.10.0 for cold migration and as a Developer Preview feature for warm migration. It is not available for shared disks for either cold or warm migration For more information, see xref:about-storage-copy-offload_vmware[About migrating {vmw} virtual machines by using storage copy offload].
```

Comment:
This feature makes use of the command `vmkfstools` on the ESXi which invokes XCOPY on the storage array using the iSCSI/FC connection